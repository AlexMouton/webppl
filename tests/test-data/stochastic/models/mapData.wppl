var model = function() {
  return mapData({data: [.6, .1]}, flip);
};
