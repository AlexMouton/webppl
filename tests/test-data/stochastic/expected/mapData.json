{
  "hist": {
    "[false,false]": 0.36,
    "[false,true]": 0.04,
    "[true,false]": 0.54,
    "[true,true]": 0.06
  }
}
